e2e_playwright/st_audio_test.py

```diff
@@ -61,3 +61,56 @@ def test_audio_end_time_loop(app: Page):
     app.wait_for_timeout(6000)
     expect(audio_element).to_have_js_property("paused", False)
     wait_until(app, lambda: 16 < audio_element.evaluate("el => el.currentTime") < 18)
+
+
+@pytest.mark.skip_browser("webkit")
```

Comment:
nit: do we still need to skip `webkit` here since the `test_audio_remount_no_autoplay` test seems to work fine without skipping webkit?

Reply:
You're right, I'll remove that

Reply:
Fixed on 264e547 👍